models/Version.php

```diff
@@ -366,7 +368,11 @@ public function delete()
     public function loadData($renewReferences = true)
     {
         $storage = Storage::get('version');
-        $data = $storage->read($this->getStorageFilename());
+        try {
+            $data = $storage->read($this->getStorageFilename());
+        } catch(UnableToReadFile | FilesystemException $e) {
```

Comment:
I don't think it's necessary to add both here, because the `UnableToReadFile` is already a `FilesystemException`, no?

Reply:
Well okay, if Flysystem actually mentions both... maybe we should leave it as is...

Reply:
It would be good to know when which exception is caused. Maybe we shouldn't always delete the version if the error is only temporary.

Reply:
In the docs there is only one: https://flysystem.thephpleague.com/v2/docs/usage/exception-handling/

Reply:
Changed it to only catch `UnableToReadFile`